### PR TITLE
css-markdown

### PR DESCRIPTION
**Pull request by stutirao**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

**Comment by satyamyadav** [comment]
@stutirao  Please use a tabular format or tab space indentation for variables. For example:
**Table:**
```
| token 	| value  	| preview 	|
|-	        |-	        |-	        | 
| --jal 	|`#0070DD`| ![#0070DD](https://via.placeholder.com/15/0070DD/000000?text=+) 	|

```
| token 	| value  	| preview 	|
|-	      |-	      |-	        | 
| --jal 	|`#0070DD`| #0070DD 	|

**Tab space:**

```
`--jal` &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `#0070DD`| ![#0070DD](https://via.placeholder.com/15/0070DD/000000?text=+)

```
`--jal` &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `#0070DD`

**Comment by satyamyadav** [comment]
@stutirao  Please make list only for utility classes. 

Screenshot 2020-07-07 at 1 09 46 PM

File: css/Readme.md [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+| **6** | margin: --spacing-2 ; |
+| **7** | margin: --spacing-xl ; |
+| **8** | margin: --spacing-3 ; |
+| **9** | margin: --spacing-4 ; |
+| **10** | margin: --spacing-5 ; |
+| **11** | margin: --spacing-6 ; |
+| **12** | margin: --spacing-7 ; |
+| **13** | margin: --spacing-8 ; |
+| **14** | margin: --spacing-9 ; |
+| **auto** | margin: auto ; |
```
Comment (aditya-kumawat) [inline css/Readme.md]: replace `margin: --spacing-xs ;` with `--spacing-xs`. Remove margin from every row.

File: css/Readme.md [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+| d-table-row | display: table-row ; | 
+| d-table-cell | display: table-cell ; | 
+| d-flex | display: flex ; | 
+| d-inline-flex | display: inline-flex ;| 
+
+# UTILITY 
+
+| value | properties |
+| ----- | ---------- |
+| hide-scroll-bar::-webkit-scrollbar | height: 0px;<br> /* Remove scrollbar space */<br>background: transparent;<br> /*Optional: just make scrollbar invisible */ |
```
Comment (aditya-kumawat) [inline css/Readme.md]: Remove comments

File: css/Readme.md [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+| hide-scroll-bar | -ms-overflow-style: none;<br>scrollbar-width: none;<br> |
+| ellipsis | white-space: nowrap;<br>overflow: hidden;<br>text-overflow: ellipsis;<br> |
+
+
+## SHADOW
+  | value | properties |
```
Comment (aditya-kumawat) [inline css/Readme.md:404]: Mention what color-mod does

File: css/Readme.md [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+## GRID BREAKPOINTS
+  ##### Define the minimum dimensions at which your layout will change,
+  ##### adapting to different screen sizes, for use in media queries.
+
+  | value | properties |
+  | ----- | ---------- |
+  | --grid-s-breakpoint | 576px; |
+  | --grid-m-breakpoint | 768px; |
+  | --grid-l-breakpoint | 992px; |
+  | --grid-xl-breakpoint | 1200px; |
```
Comment (aditya-kumawat) [inline css/Readme.md]: Remove this

**Pull request merged**